SparkPost/2web2ui
# FE-1095 - Update @testing-library dependencies

[FE-1095](https://jira.int.messagesystems.com/browse/FE-1095)

### What Changed
- Updates libraries outlined in the tickets
- Removes all `query*` queries from Cypress tests
- Removed a couple of RTL tests that were causing problems following the `@testing-library/user-event` upgrade - rather than going down a huge rabbit hole, I realized these tests were duplicated by Cypress

### How To Test
- Verify packages were updated to their latest versions
- Verify tests continue to pass

### To Do
